src/routes/AuthenticatedRoutes.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect } from 'react';
+import { useLocation, Outlet, Navigate } from 'react-router-dom';
+import { makeToast } from '../components';
+
+const AuthenticatedRoutes = () => {
+  const isAuth = JSON.parse(localStorage.getItem('isAuth'));
+  const location = useLocation();
```

Comment:
the name AuthenticatedRoutes does not look good to me. What about something like RequireAuth?